activerecord/lib/active_record/relation.rb

```diff
@@ -647,7 +647,12 @@ def delete_all
 
     # Finds and destroys all records matching the specified conditions.
     # This is short-hand for <tt>relation.where(condition).destroy_all</tt>.
-    # Returns the collection of objects that were destroyed.
+    # Returns the collection of objects that were destroyed if
+    # +config.active_record.destroy_all_in_batches+ is set to +false+.
+    #
+    # If +config.active_record.destroy_all_in_batches+ is set to +true+, it will ensure
+    # to perform the record's deletion in batches
```

Comment:
I'm reading this as meaning to be plural, but not sure 😄 
```suggestion
    # to perform the records deletion in batches
```